text/0000-nft-standard.md

```diff
@@ -0,0 +1,133 @@
+- Feature Name: nft_standard
+- Start Date: 2021-08-02
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Pedro Ferreira <pedro@hathor.network>
+
+# Summary
+[summary]: #summary
+
+This document presents a standard to be followed when creating an NFT transaction and its metadata. The idea behind having a standard is to be able to identify NFT tokens and to be easy for any plataform that would like to integrate Hathor NFTs.
+
+# Motivation
+[motivation]: #motivation
+
+Creating an NFT standard is important mainly for 3 reasons:
+
+1. There are some requirements that an NFT token must fulfill in order to have its digital asset shown in our explorer, so it's important that they are described here.
+1. Having a similar structure in most NFTs (also for its metadatas) facilitates the integration with any platform to list and show information about NFTs created on Hathor. 
+1. Identifying an NFT is important to show specific information in the explorer and wallets.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Non-fungible tokens (NFTs) are unique, indivisible digital assets created on blockchains. HTR token, for instance, is a fungible token and 1 HTR of mine can be exchanged with 1 HTR of anyone without difference because all tokens are equal. Each NFT’s uniqueness can be proven by a unique identifier.
+
+In Hathor Network an NFT is a new custom token created and its unique identifier is the transaction hash.
+
+## Transaction standard
+
+A transaction can be identified as an NFT creation if it has the following structure:
+
+1. Transaction version is 2 (TOKEN_CREATION_TRANSACTION).
+1. All inputs are HTR inputs.
+1. First output has HTR value and the script is a pushdata with the NFT data followed by a OP_CHECKSIG.
+1. We can have 1 or 0 melt authority output.
```

Comment:
Suggestion: It can optionally have a melt authority output (but never more than one).